From c5474ef41329a433907f44188e2a42cb2d2806af Mon Sep 17 00:00:00 2001
From: Joern Rennecke <joern.rennecke@embecosm.com>
Date: Tue, 30 Jul 2013 16:19:59 +0100
Subject: [PATCH 02/33] ChangeLog.epiphany: Add reference to PR
 rtl-optimization/58029

---
 gcc/ChangeLog.epiphany | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gcc/ChangeLog.epiphany b/gcc/ChangeLog.epiphany
index 62f8185672c..891b777d05b 100644
--- a/gcc/ChangeLog.epiphany
+++ b/gcc/ChangeLog.epiphany
@@ -1,5 +1,6 @@
 2013-07-30  Joern Rennecke <joern.rennecke@embecosm.com>
 
+	PR rtl-optimization/58029
 	* alias.c (base_alias_check): Allow for aliases between stack-
 	and frame-pointer referenced memory.
 
-- 
2.31.1

